packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -5,7 +5,8 @@
 
 'use strict';
 
-const { commitTypes, transform } = require( '../changelog/writer-options' );
+const { availableCommitTypes } = require( './transform-commit-utils' );
+const transformCommitForCKEditor5Package = require( './transformcommitforckeditor5package' );
```

Comment:
>  do we need to parametrize the transformCommit?

How do you want to generate changelog for the `ckeditor5-dev` package?

Because we talked about it via Slack, I know what we need to do. If you have a batter idea, let me know.